core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -402,7 +402,7 @@ public void run() {
                             // connect to the node, see if we manage to do it, if not, bail
                             if (!nodeFoundByAddress) {
                                 logger.trace("[{}] connecting (light) to {}", sendPingsHandler.id(), finalNodeToSend);
```

Comment:
Might be worth logging the `(light with handshake)`? Since the other one is _not_ light, it really doesn't matter to me though.